gpu/gosl/slrand/slrand.wgsl

```diff
@@ -0,0 +1,201 @@
+// Copyright (c) 2024, Cogent Core. All rights reserved.
```

Comment:
So all of these `.wgsl` files are generated? What about indicating such with a `DO NOT EDIT` comment?

Reply:
we can add that later.  this actual one is not auto-generated btw.